.travis.yml

```diff
@@ -38,7 +38,3 @@ script:
 # build from tarball
     - make distdir=dist dist && tar xvf dist.tar.gz && cd dist
     - ./configure --prefix=/ --libdir=/lib --host=$CHOST --build=x86_64-pc-linux-gnu --program-prefix='' $CONFIG && make
-    
-cache:
-  directories:
-  - download-cache
```

Comment:
Removing the cache will make travis runs take much longer as it has to build openssl each time. As it is travis has been failing often lately because of timeouts and such errors unrelated to our code base. So I think we should try harder to get the cache working again.

Reply:
we do have just "download" cache. I added it back.

you are right, we can implement "build" cache as well. I will do it in future PR